lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -74,7 +74,8 @@ def platform_options
       OPT_NAMES.each do |name|
         options[name] = @platform[name] unless @platform[name].nil?
       end
-      options
+      share_host_directory = @project.options.common.share_host_directory || @project.options.test.share_host_directory
+      options.merge({ 'share_host_directory' => share_host_directory })
```

Comment:
This should not be added here as it is not a platform option (that can be defined with a JSON file in `lib/engines/hcktest/platforms` or with `platforms_defaults` in `lib/setupmanagers/qemuhck/qemu_machine.json`).